nmmo/entity/entity.py

```diff
@@ -296,7 +296,8 @@ def receive_damage(self, source, dmg):
     if source is None or not source.is_player: # nobody or npcs cannot loot
       if self.config.ITEM_SYSTEM_ENABLED:
         for item in list(self.inventory.items):
-          item.destroy()
+          self.inventory.remove(item) # delete from the inventory/market
+          item.destroy() # delete from the datastore
```

Comment:
shouldn't destroy() handle the removal from inventory/market?

Reply:
I looked into it, and I will create a separate PR for this.